lib/channel.js

```diff
@@ -0,0 +1,66 @@
+var WebSocketLib = require('ws');
+var util         = require('./util');
+
+function createChannel(port) {
```

Comment:
Yeah, this is my personal preference to doing most oop in js, there's no real need to use prototypes here, most objects don't get instantiated more than once, especially not a websocket object.

Want me to change it to your method? I'll use those in the future too then.